locale/mr.js

```diff
@@ -1,6 +1,7 @@
 //! moment.js locale configuration
```

Comment:
I'm new to GitHub and couldn't find a way to remove the file from PR. Can you please guide me to do the needful?